dropshot/Cargo.toml

```diff
@@ -37,6 +38,7 @@ slog-json = "2.6.1"
 slog-term = "2.9.0"
 tokio-rustls = "0.24.0"
 toml = "0.7.4"
+waitgroup = "0.1.2"
```

Comment:
This is fine.  I see this pulls in two deps.  It's a shame if there's nothing in tokio or futures that can already do this. :shrug: